client/src/index.js

```diff
@@ -17,6 +18,7 @@ export default function App() {
                     <Route index element={<Home />} />
                     <Route path="task-1" element={<Task1 />} />
                     <Route path="task-2" element={<Task2 />} />
+                    <Route path="TelemComponent" element={<TelemCompoment />} />
```

Comment:
Your added "TelemCompoment" should be added to the Task1.js and Task2.js component

Also typo in Componment, update. the file name and its usages to Component